integration/tests/cook/test_cli.py

```diff
@@ -623,6 +627,10 @@ def test_ssh_job_uuid(self):
         self.assertIn(f'-t {hostname} cd', stdout)
         self.assertIn('; bash', stdout)
 
+    @unittest.skipIf(util.has_ephemeral_hosts(), 'If the cluster under test has ephemeral hosts, then it is generally '
+                                                 'a bad idea to use HOSTNAME EQUALS constraints, because it can cause '
+                                                 'the process responsible for launching hosts to launch hosts that '
+                                                 'never get used')
     def test_ssh_no_instances(self):
         raw_job = {'command': 'ls', 'constraints': [['HOSTNAME', 'EQUALS', 'will not get scheduled']]}
```

Comment:
Is it possible to use a different constraint that causes jobs to not be scheduled and avoid the issue of launching hosts that never get used?

Reply:
The CLI tests don't currently support docker jobs anyway, so I removed the `skipIf` until we address that in a separate PR